From 14f88afc5170d6c157d9e793f6f438ad1d7987a7 Mon Sep 17 00:00:00 2001
From: Vishal Sagar <vishal.sagar@xilinx.com>
Date: Mon, 10 Aug 2020 09:32:11 -0700
Subject: [PATCH 1514/1852] dt-bindings: media: xilinx: csi2rxss: Add YUV 420
 8bits support

commit 701455d4cf53f911fa3192e52382eedb072769bc from
https://github.com/Xilinx/linux-xlnx.git

This patch adds the new string for YUV 420 8bits (0x18 data type)
support.

Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 .../devicetree/bindings/media/xilinx/xlnx,csi2rxss.txt         | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Documentation/devicetree/bindings/media/xilinx/xlnx,csi2rxss.txt b/Documentation/devicetree/bindings/media/xilinx/xlnx,csi2rxss.txt
index 0993ad622008..a181a5aa0ed1 100644
--- a/Documentation/devicetree/bindings/media/xilinx/xlnx,csi2rxss.txt
+++ b/Documentation/devicetree/bindings/media/xilinx/xlnx,csi2rxss.txt
@@ -32,7 +32,8 @@ Required properties:
 - xlnx,csi-pxl-format: This denotes the CSI Data type selected in hw design.
   Packets other than this data type (except for RAW8 and User defined data
   types) will be filtered out. Possible values are RAW6, RAW7, RAW8, RAW10,
-  RAW12, RAW14, RAW16, RAW20, RGB444, RGB555, RGB565, RGB666, RGB888 and YUV4228bit.
+  RAW12, RAW14, RAW16, RAW20, RGB444, RGB555, RGB565, RGB666, RGB888,
+  YUV4228bit and YUV420_8bit.
 
 - xlnx,vfb: Video Format Bridge, Denotes if Video Format Bridge is selected
   so that output is as per AXI stream documented in UG934.
-- 
2.31.1

